components/gcp/dataproc/submit_sparksql_job/component.yaml

```diff
@@ -75,7 +75,7 @@ outputs:
     type: UI metadata
 implementation:
   container:
-    image: gcr.io/ml-pipeline/ml-pipeline-gcp:ad9bd5648dd0453005225779f25d8cebebc7ca00
+    image: gcr.io/ml-pipeline/ml-pipeline-gcp:1.0.0-rc.1
```

Comment:
There have been no changes to this image. Can we avoid changing the component and references?
We've got feedback from a customer that it's hard to track component updates and understand what has changed.

Reply:
I've summarized my opinions on the email thread. Would you mind replying there so we have a centralized discussion? It's still very easy to change this to optional now.